### ISSUE TITLE
PHI-META-001: replace Phi-stability confidence with prospective metacognitive calibration

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#5413]
Parent: #5401
Depends on semantic observation work: #5403
Related control work: #5412
Related expression/authority firewall: #5385

## Finding

Symthaea currently writes the shared-looking concept `metacognitive_confidence` from at least two materially different mechanisms.

### Producer A — Phi stability heuristic

`MetacognitiveSubsystem` keeps a rolling `phi_history` and computes:

```text
variance(phi_history)
confidence = 1 / (1 + 10 * variance)
```

then assigns:

```text
state.metacognitive_confidence = confidence
```

It also defines `metacognitive_coherence` as proximity between `state.phi` and `state.consciousness_level`, and extrapolates `predicted_phi` from the Phi trend.

Therefore current semantics include:

```text
stable Phi trajectory -> high metacognitive confidence
```

but stability does not establish task correctness or future reliability.

### Producer B — self-model fit

`MetaConsciousness` assigns:

```text
metacognitive_confidence = self_model.confidence
```

where `SelfModel::update()` first folds the current observed state/Phi into the self-model and then computes a prediction/reconstruction error against that same current observation.

Therefore the quantity is closer to **post-update self-model fit/reconstruction consistency** than a prospective probability that a future answer/action will be correct.

### Additional implementation mismatch

`MetaConfig.self_model_learning_rate` is modified by `meta_learn()`, but the current `SelfModel::update()` uses a hard-coded 0.8/0.2 smoothing rule and does not consume that configuration value.

Thus:

```text
meta-learning changes configured self-model learning rate
!= demonstrated change to SelfModel::update behavior
```

This should be characterized/fixed separately from confidence calibration rather than using the config field as evidence that the self-model actually adapted its learning rate.

## Research basis

Modern metacognition work distinguishes confidence from accuracy and emphasizes the quality of the confidence-performance relationship. Relevant concepts include:

```text
metacognitive bias
metacognitive sensitivity
metacognitive efficiency
confidence calibration
```

See, for example:

- Fleming, *Metacognition and Confidence: A Review and Synthesis*, Annual Review of Psychology 75 (2024), DOI `10.1146/annurev-psych-022423-032425`;
- Fleming & Lau, *How to measure metacognition*, Frontiers in Human Neuroscience (2014), PMID `25076880`.

These sources motivate measurement design; they do not validate any particular Symthaea confidence model.

## Core theorem

```text
internal stability
!= correctness probability

self-model reconstruction fit
!= correctness probability

prospectively calibrated confidence
!= truth or authority
```

## Goal

Separate internal diagnostic observations from an explicit, prospectively scored **confidence forecast about a defined future outcome**.

## M0 — producer/consumer inventory

Freeze every production writer/reader of at least:

```text
metacognitive_confidence
metacognitive_coherence
self_model.confidence
predicted_phi
phi_trend
```

Classify each writer by actual proposition, for example:

```text
InternalStabilityScore
SelfModelFitScore
ProspectiveOutcomeConfidence
LegacyComposite
Unknown
```

Do not preserve a single semantic label merely because the values are all `[0,1]`.

## M1 — typed diagnostic quantities

Use #5403-style source/quantity identities to preserve internal signals honestly, for example conceptually:

```text
PhiStabilityObservation
SelfModelFitObservation
PhiTrendObservation
IntegrationCoherenceObservation
```

These are candidate features for metacognitive forecasting, not confidence estimates by themselves.

## M2 — prospective confidence receipt

Introduce a small record created **before the outcome is observed**, conceptually:

```text
MetacognitiveForecastV1 {
  forecast_id,
  target/outcome_profile,
  subject/task identity,
  model/profile identity,
  input snapshot/evidence refs,
  predicted_probability,
  created_before_outcome,
}
```

The target must be explicit, e.g.:

```text
answer_correct_under_exact_evaluator
plan_completes_under_exact_test_suite
next_step_error_below_threshold
self_correction_succeeds
```

Avoid generic `confidence: f64` without a proposition/outcome target.

## M3 — outcome binding and calibration receipt

After the outcome becomes known, bind it separately:

```text
MetacognitiveOutcomeV1
CalibrationReceiptV1
```

No retrospective rewriting of the original forecast.

Preserve failed/incorrect predictions.

## M4 — baselines and candidate features

Compare at least:

```text
constant/base-rate predictor
recent empirical accuracy
Phi level
Phi variance/stability
Phi trend
self-model fit
prediction error
model-native/task confidence if available
integration/segregation vector from later #5412 work
combined learned/calibrated model
```

Use held-out tasks/seeds/time periods so the evaluator is not scored on the same samples used to tune it.

## M5 — scoring

At minimum report appropriate prospective metrics such as:

```text
Brier score
log loss
calibration error / reliability curve
selective accuracy or risk-coverage
failure-detection AUROC where appropriate
```

Where useful, also distinguish metacognitive bias/sensitivity/efficiency instead of collapsing them to one score.

Do not select the model solely on one in-sample metric.

## M6 — self-model update repair

Independently repair/characterize the current self-model learning-rate mismatch.

Either:

```text
SelfModel::update consumes the declared learning-rate/profile
```

or narrow/remove the unused `self_model_learning_rate` / meta-learning claim.

Require an actual behavioral test proving that changing the configured rate changes the update trajectory before calling it adaptive meta-learning.

## M7 — control integration

Only after prospective calibration demonstrates utility may the forecast guide resource policy such as:

```text
verification budget
additional tool checks
ask/abstain thresholds
exploration depth
second-model/critic invocation
```

Use monotonic safety semantics where practical:

```text
lower calibrated confidence -> may increase caution/resources
higher calibrated confidence -> does not exceed evidence/action authority ceilings
```

Coordinate expression behavior with #5385.

## M8 — no authority laundering

Even a strongly calibrated forecast remains advisory evidence about expected performance.

```text
P(correct)=0.95
!= proposition established true
!= cryptographic verification
!= causal proof
!= authorization
```

## Required tests

- constant Phi gives high stability score but does not automatically yield high prospective confidence;
- a stable-but-wrong fixture is retained and penalizes calibration;
- a noisy-but-accurate fixture can outperform Phi stability;
- forecasts are immutable once outcome evaluation begins;
- forecast target/profile is mandatory;
- self-model fit and prospective confidence cannot be implicitly converted into each other;
- changing `self_model_learning_rate` demonstrably changes behavior or the field/claim is removed/narrowed;
- no calibrated forecast can mint MEL-EPI claim authority or action authority by conversion.

## Claim ceiling

A successful result may establish:

```text
forecast model M is calibrated/predictive for outcome Y
under task/profile P
on held-out evaluation E
```

It does not establish:

```text
Symthaea knows that an answer is true
Phi measures correctness
consciousness
universal metacognitive competence
execution/governance authority
```

The purpose is to make internal telemetry genuinely useful for self-monitoring by tying confidence to future outcomes rather than to internal stability alone.
